drivers/counter/counter_mcux_lpc_rtc.c

```diff
@@ -132,15 +132,22 @@ static int mcux_lpc_rtc_set_top_value(const struct device *dev,
 	const struct mcux_lpc_rtc_config *config =
 			CONTAINER_OF(info, struct mcux_lpc_rtc_config, info);
 	struct mcux_lpc_rtc_data *data = dev->data;
+	uint32_t enable = (config->base->CTRL & RTC_CTRL_RTC_EN_MASK);
 
 	if (cfg->ticks != info->max_top_value) {
-		LOG_ERR("Wrap can only be set to 0x%x.", info->max_top_value);
+		LOG_DBG("Wrap can only be set to 0x%x.", info->max_top_value);
```

Comment:
I think this should remain as LOG_ERR because it is explaining why a function is exiting with an error return.

Reply:
This shows up repeatedly when running the power manager tests. User can adjust the debug level in case of issues.

Reply:
it should at least be a warning

Reply:
actually, why are we even implementing this function, it's optional in the API and it sounds like this hardware doesn't support it